solidity/scripts/check_balance.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from brownie import USDCToken, accounts
+
+
+L1_TOKEN_ADDRESS = os.environ.get("PARACLEAR_L1_TOKEN_ADDRESS")
+USER_PRIVATE_KEY = os.environ.get("USER_PRIVATE_KEY")
+
+
+def main():
+    """
+    Add funds to user accounts.
+    Requirement: All accounts must be preloaded on brownie
+    before running the deploy script.
+    """
+
+    user_account = accounts.add(USER_PRIVATE_KEY)
+
+    from_user_account = {"from": user_account}
+    usdc = USDCToken.at(L1_TOKEN_ADDRESS)
+
+    tx = usdc.balanceOf(user_account, from_user_account)
+
+    print("new balance ", tx)
```

Comment:
we can return quantum / decimal_places converted balance for easier interpretation 
`bal * 10 ** (TOKEN_QUANTUMS * -1)`